appveyor.yml

```diff
@@ -84,8 +93,14 @@ after_build:
   # Show nupic installation folder by trying to import nupic, if works, it prints
   # the absolute path of nupic.__file__, which the installation folder itself.
   - python -c "import sys;import os;import nupic.data;sys.stdout.write(os.path.abspath(os.path.join(nupic.data.__file__, '../..')) + '\n')"
+
+  # Setup local USER name
   - set USER=appveyor
 
+  # Copy a custom AppVeyor site XML for OPF database connections,
+  # into local USER site-packages directory
+  - copy ci\appveyor\nupic-site.xml C:\Users\appveyor\AppData\Roaming\Python\Python27\site-packages\nupic\support\nupic-site.xml
```

Comment:
Can we just use `src/nupic/support/nupic-default.xml`? Why create a copy of that at `ci/appveyor/nupic-site.xml`?